### PR TITLE
docs: filtering vulnerabilities with trivy

### LINKED ISSUE #214: [DOC] filtering vulns with trivy
### What kind of documentation improvement is needed?

None

### What is the change that is needed?

document how to filter vulns using rego policy or ignore file
https://aquasecurity.github.io/trivy/v0.43/docs/configuration/filtering/

example from CI: https://github.com/project-copacetic/copacetic/blob/3e22d49509ef9ef16aeedccfcc050f4ad627ed92/.github/workflows/build.yml#L112-L126

### PR DESCRIPTION
**Pull request by anubhav06**
Added docs for filtering vulnerabilities with Trivy

Closes #214

### CODE REVIEW

File: website/docs/quick-start.md [hunk 1]
```diff
@@ -131,4 +131,65 @@ This sample illustrates how to patch containers using vulnerability reports with
     2022/05/16 18:00:17 [notice] 31#31: signal 28 (SIGWINCH) received
     ```
 
-   You can stop the container by opening a new shell instance and running: `docker stop nginx-test`
\ No newline at end of file
+   You can stop the container by opening a new shell instance and running: `docker stop nginx-test`
+
+## Filtering Vulnerabilities
```
Comment (sozercan) [inline website/docs/quick-start.md:136]: can we create a new top level bullet (similar to quick start - see https://github.com/project-copacetic/copacetic/blob/main/website/sidebars.js) called `Troubleshooting` or similar and add this content there?

File: website/docs/quick-start.md [hunk 2]
```diff
@@ -131,4 +131,65 @@ This sample illustrates how to patch containers using vulnerability reports with
...
+### Ignore File
+
+Use a `.trivyignore` file to list all the vulnerabilities you want to ignore.
+
+Example:
+```bash
+$ cat .trivyignore
+
+# Accept the risk
+CVE-2018-14618
```
Comment (sozercan) [inline website/docs/quick-start.md:191]:
```suggestion
CVE-2018-14618
```

File: website/docs/quick-start.md [hunk 3]
```diff
@@ -131,4 +131,65 @@ This sample illustrates how to patch containers using vulnerability reports with
...
+Use a `.trivyignore` file to list all the vulnerabilities you want to ignore.
+
+Example:
+```bash
+$ cat .trivyignore
+
+# Accept the risk
+CVE-2018-14618    
+```
+In the above example, the vulnerability  CVE-2018-14618 is ignored while scanning, and hence while patching the image.
```
Comment (sozercan) [inline website/docs/quick-start.md:193]:
```suggestion
In the above example, the vulnerability CVE-2018-14618 is ignored while scanning, and hence while patching the image.
```

**Review by sozercan — CHANGES_REQUESTED** [review]

File: website/docs/quick-start.md (hunk 1 above)
Comment (anubhav06) [reply website/docs/quick-start.md:136]: Yess, that will be perfect!
Thanks, will do that

**Comment by sozercan** [comment]
ah yes, looks like that was a bug. it should be `v0.1.x`

**Comment by sozercan** [comment]
@anubhav06 can you update https://github.com/project-copacetic/copacetic/blob/main/website/versioned_sidebars/version-v0.1.0-sidebars.json too?

**Review by sozercan — APPROVED** [review]
Thank you! LGTM

**Pull request merged**